src/DisplayData.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace TheRealMVP;
+
+class DisplayData
+{    
+    /**
+     * Iterate through each team object to display values and inject html
+     *
+     * @param  mixed $data
+     * 
+     * @return string
+     */
+    public function displayAllTeams(array $data) : string
```

Comment:
Because this method never needs to access any properties on the object (basically doesn't need $this) you can make this into a static method.

This makes accessing it quicker/easier because you don't need to instantiate a DisplayData object in order to display the teams.